src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/services/KafkaService.scala

```diff
@@ -50,6 +53,14 @@ class KafkaServiceImpl @Inject() (generalConfig: GeneralConfig) extends KafkaSer
     getOffsets(topic, consumerProperties, EndOffsets)
   }
 
+  def getOffsets(topic: String, consumerProperties: Properties): BeginningEndOffsets = {
```

Comment:
I would rename this to `getBeginningEndOffsets` to be consistent

Reply:
Done